hawkbit-rest/hawkbit-mgmt-resource/src/main/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtTargetResource.java

```diff
@@ -66,13 +66,17 @@
 
     private final TargetManagement targetManagement;
 
+    private final TargetQueryExecutionManagement targetQueryExecutionManagement;
```

Comment:
`TargetQueryExecutionManagement` is always used as a raw type. Do you really need the type `T` on interface level? If so type parameter should be specified properly by consumers of this class

Reply:
It's your sonar-rule which avoids other solutions...